packages/flame/test/sprite_test.dart

```diff
@@ -0,0 +1,50 @@
+import 'dart:ui';
+
+import 'package:flame/components.dart';
+import 'package:flame/game.dart';
+import 'package:flame_test/flame_test.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import '_assets/images.dart';
+
+void main() {
+  group('Sprite', () {
+    FlameTester(() => FlameGame()).testGameWidget(
+      'Render with anchor',
+      setUp: (game, tester) async {
+        game.add(MyComponent()..position = Vector2.all(25));
+        await game.ready();
+      },
+      verify: (game, tester) async {
+        await expectLater(
+          find.byGame<FlameGame>(),
+          matchesGoldenFile('_goldens/sprite_test_1.png'),
+        );
+      },
+    );
+  });
+}
+
+class MyComponent extends PositionComponent {
+  MyComponent() : super(size: Vector2(200, 400));
+  late final Sprite sprite;
+
+  @override
+  Future<void> onLoad() async {
+    sprite = Sprite(await loadFlameImage());
+  }
+
+  @override
+  void render(Canvas canvas) {
+    canvas.drawRect(
+      Rect.fromLTRB(0, 0, size.x, size.y),
```

Comment:
I think there was an ext function to convert size to rect, not sure though

Reply:
You're right, it's `size.toRect()`